doc/sphinx/proofs/writing-proofs/equality.rst

```diff
@@ -189,22 +189,24 @@ Rewriting with Leibniz and setoid equality
 
       .. coqtop:: none
 
-         Require Import Arith.
+         From Stdlib Require Import Setoid.
+
+         Axiom add_comm : forall n m, n + m = m + n.
```

Comment:
If the documentation is built without Stdlib, then the hint databases for proof search tactics will be fairly empty.  That may make it very tricky to add/update proof search examples--something I've been working on for while.  One would have to figure out what hints are missing, then recreate them as you do add_comm.  I still believe Stdlib should be included for building the doc.

Reply:
> If the documentation is built without Stdlib, then the hint databases for proof search tactics will be fairly empty. That may make it very tricky to add/update proof search examples--something I've been working on for while. One would have to figure out what hints are missing, then recreate the hints as you do add_comm. I still believe Stdlib should be included for building the doc.

Your request to keep depending on the Stdlib was taken into account. It just to avoid adding `extra` to this example that Pierre modified it.

Reply:
OK.  My comment is not closely tied to these lines, I just wanted to include in as part of a review rather than as single comment.  I hope I won't have to recreate a bunch of hints for proof search examples.